vendor/github.com/danwinship/knftables/README.md

```diff
@@ -0,0 +1,177 @@
+# knftables: a golang nftables library
```

Comment:
curious what the "k" stands for?

Reply:
"kool"? :sunglasses: 

It stands for "kubernetes-adjacent".

Originally it was just "nftables" but Antonio pointed out that having a generic name like that can make it hard to refer to, both in code (files that need to import both `"github.com/danwinship/nftables"` and `"k8s.io/kubernetes/pkg/proxy/nftables"` would have to rename one of them) and in conversation (compare https://github.com/cilium/ebpf/discussions/798)